src/components/templates/OrderCompleteTemplate.jsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable-next-line no-unsafe-optional-chaining */
+import { useMutation } from '@tanstack/react-query';
+import { getOrderFromId } from '../../services/order';
+import { comma } from '../../utils/convert';
+import { useNavigate } from 'react-router-dom';
+
+const OrderCompleteTemplate = ({ data }) => {
+    // 사용자의 장바구니 목록을 조회해서 보여주는 것
+
+    const { products, totalPrice } = data.data.response || undefined;
+
+    const OrderItems = () => {
+        let renderComponent = [];
+        if (Array.isArray(products) === false) return;
```

Comment:
```suggestion
        if (!Array.isArray(products)) return;
```

처럼 비교를 안하고 단축해서 쓸 수 있겠네요!